docs/src/references/c8y-configuration-management.md

```diff
@@ -274,107 +274,125 @@ The HTTP filetransfer feature of the `tedge_agent` provides the service to trans
     * a temporary file can be uploaded with an HTTP PUT request to `http://<thin-edge IP address>/tedge/tmpfiles`,
       where the HTTP response contains a _temporary URL_.
     * a temporary file can be downloaded with an HTTP GET request to `http://<ip address of thin-edge device>/<temporary URL>`
-    * a local API allows the plugin to access (expose and obtains) a temporary file directly via the thin-edge device's local filesystem
 
-    * TODO: Investigate and decide about filetransfer's local API in scheduled prototype (https://github.com/thin-edge/thin-edge.io/issues/1307), according to topics below:
- 
-      * local API must avoid that there are two copies of the same file at any point in time
-      * local API must avoid that the file content is transfered/copied from one to another place, when the plugin _obtains_ the file. 
-        Instead just the directory link may change (as it is the case for bash `mv` command).
+## Fetching configuration file from child device to cloud
 
+A configuration file snapshot is fetched from an external device as illustrated below.
 
-## Fetching configuration file from child device to cloud
+```mermaid
+sequenceDiagram
+    participant External Device
```

Comment:
```suggestion
    participant Child Device
```

Reply:
I used here intentionally the term 'External Device', to make clear that we have in that case not a local process, but an external device.

Reply:
> I used here intentionally the term 'External Device', to make clear that we have in that case not a local process, but an external device.

But this flow is valid for external as well as local child device processes/containers as well, right? We don't have a different flow for local child device processes, do we?

Reply:
> I think we need a clear description what is "external device" and `child device" to understand the difference.

There is now a terminology section that describes all used terms on context of child-devices, see https://github.com/thin-edge/thin-edge.io/pull/1236/files#diff-2712a2328b3177d213f13fff299592fad26c8defd1c0b704400b93c064d95ba8